main/admin/session_fields.php

```diff
@@ -0,0 +1,204 @@
+<?php
```

Comment:
The same as above. No need to create this file:

main/admin/extra_fields.php?type=session